netci.groovy

```diff
@@ -122,7 +129,7 @@ def static isLongGc(def scenario) {
 
 def static setTestJobTimeOut(newJob, scenario) {
     if (isGCStressRelatedTesting(scenario)) {
-        Utilities.setJobTimeout(newJob, 1440)
+        Utilities.setJobTimeout(newJob, 2880)
```

Comment:
I talked to @mmitche and he suggested 48 hours. I'd just see how this goes and adjust the timeout appropriately. The problem with big timeouts in the lab is that if at any point we get a deadlock or a hung process it takes long time to get the issue to service and keeps machines in the lab occupied. But again, I don't feel strong at all about it. If you think it is something we should do, just say so and I'd be happy to revise the PR.;